content/millennium/r4/base/workflow/appointment.md

```diff
@@ -51,11 +51,26 @@ The following fields are returned if valued:
 * [Comment](http://hl7.org/fhir/R4/appointment-definitions.html#Appointment.comment){:target="_blank"}
```

Comment:
Can we correct line 37 above to be"Cancelation Reason" instead of "Cancellation Reason"